src/Routes/Register/NewRegister.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
Why make a <NewRegister> component and not just replace the current one all together? Is this gonna be hidden behind a FF? And if so it needs to be slapped in there

Reply:
Mainly because this is a completely different design and it's not complete in this PR.